training-net/src/appsettings.json

```diff
@@ -4,5 +4,8 @@
       "Default": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*" ,
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=(localdb)\\db-training-net;Database=db-training-net;Trusted_Connection=True;MultipleActiveResultSets=true"

```

Comment:
is this working? `services.AddDbContext<DataBaseContext>(options =>  options.UseNpgsql(Configuration["ConnectionString"]));` in this line you are using a var called "ConnectionString" and it isn't exist on this json. also, i dont understand what "DefaultConnection" is